lm_eval/tasks/VeryfIT/veryfIT_full.yaml

```diff
@@ -0,0 +1,22 @@
+tag:
+  - fact-checking
+  - factual knowledge
+dataset_path: json
+dataset_name: null
+dataset_kwargs:
+  data_files:
+    test: /home/jj/Downloads/full_subset.json
```

Comment:
this is probably local folder, so how these new benchmarks will be run on any other system other than yours?

Reply:
this is probably a no-go for PR acceptance, as it is public repo and has to be working without need for private sets

Reply:
Understandable
I talked to my supervisor, we are working for a conference https://clic2024.ilc.cnr.it/calamita/ , gathering benchmarks and running evaluations using this library. In the near future, after some work, we could push close to 40 tasks for italian.
Thank you for your attention, I'll close this PR.